### DISCUSSION TITLE
Copy To USB (CE-33116-4) Solution

### DISCUSSION THREAD

**Discussion by ghost** [bucanero/apollo-ps4#75] [Show and tell]
When i copy my save game to USB storage using "normal PS4 method", but i get the ERROR (CE-33116-4), so i tried Apollo "file backup" to copy to USB storage, but it gives weird files different than the files PS4 can read or will give.

When you use the normal method "PS4 method" what i want to do is save data from "Fpkg" to retail PS4.

**Comment by ghost** [comment]
I found the solution, i used PS4 mounter for 9.00.

## In PS4:

1- Run PS4 debug payload
2- Run FTP server

## PS4 Save Mounter (PC)

1. Connect to the PS4 
2. Click "setup"
3. Click "get games"
4. Select the "CUSAXXXXX"
5. Click "search"
6. Choose the "savedata" you want 
7. Click "mount"

### In your computer (FTP)

1. Connect to your PS4
2. Go to `/mnt/sandbox/NBSX2001_/savedata0` folder
3. Replace the "keystone" file 
3.1. if you don't know where to find the "keystone" file go down to notes
4. Click "unmount"
5. Copy the savedata from `user/home/PSN-ID/savedata`
6. Either using FTP or PS4Xplorer:
    - rename the save to remove "sdimg_" and copy it to your retail PS4 using your flash.

- f:/PS4/SAVEDATA/Account-ID/CUSAXXXXX/Savedata-Files


## Notes
I think the save can be playable again in your JB PS4 if you resign it (Apollo)

For the keystone: you need to dump the keystone from  the disc game you own, but if you don't have the time to dump go to /r/Keystones the problem it's outdated. Another solution is, you can get it from your save but it needs to be played on a original game, not FPKG.

**Reply by bucanero** [reply]
> For The Keystone
> 
> You Need To Dump The Keystone From The Disc Game You Own
> 
> And If You Don't Have The Time To Dump Go To
> 
> Reddit/r/Keystones

Regarding keystones, you can save a lot of time and use Apollo to import and export keystones as needed. When you open save with Apollo, you have two options:
- Export keystone
- Import keystone

Instead of Save mounter and doing everything manually, you can use Apollo and swap keystones to match the content.

The real problem here is that many fpkgs are built with a wrong or "blank" keystone. They're not using the original content keystone and that results in incompatible save-games. 
If users build the fpkg properly and use the original keystone, then there's no need to do all this manual work and saves can be used without modifications.

**Reply by ghost** [reply]
Thx For Explaining

**Comment by bucanero** [comment]
> When I Copy My Save Game To USB Storage Using "Normal PS4 Method" But I Get The ERROR (CE-33116-4)
> 
> So I Tried Apollo "File Backup" To Copy To USB Storage
> 
> But It Gives Weird Files Different Than The Files PS4 Can Read Or Will Give
> 

Apollo exports and copy save-games in decrypted (unlocked) format. That means that they're free from any firmware version, keystone, etc. Those are the real files loaded and saved by the game.

Note: to import decrypted saves, you need to use Apollo or other jailbreak tools like save mounter.
